src/vuestic-theme/vuestic-components/vuestic-sidebar/VuesticSidebar.vue

```diff
@@ -45,75 +45,59 @@
 </template>
 
 <script>
-  import {mapGetters, mapActions} from 'vuex'
   import Expanding from 'vue-bulma-expanding/src/Expanding'
-
   export default {
-    name: 'sidebar',
+    name: 'vuestic-sidebar',
     components: {
-      Expanding
-    },
-    methods: {
-      ...mapActions({
-        expand: 'toggleExpandMenuItem'
-      }),
-      toggleMenuItem (item) {
-        if (item.children) {
-          this.expand({   // TODO: check default prop
-            menuItem: item,
-            expand: !item.meta.expanded
-          })
-        }
-      }
-    },
-    computed: {
-      ...mapGetters({
-        'menuItems': 'menuItems'
-      })
+      Expanding,
     },
-    watch: {
-      $route (route) {
-        let parent = _findMatchingParentMenuItem.call(this, route.name) || {}
-        this.menuItems.forEach((item) => {
-          this.expand({
-            menuItem: item,
-            expand: parent.name === item.name
-          })
-        })
+    props: {
+      items: {
+        type: Array,
+        required: true
+      },
+      hidden: {
+        type: Boolean,
+        required: true,
+      },
+      toggleWithoutAnimation: {
+        type: Boolean,
+        required: true,
       }
     },
     data () {
       return {
-        show: false
+        menuItems: this.items
+      }
+    },
+    methods: {
+      onClick (item) {
+        this.$emit('toggle-item', item)
+      },
+    },
+    computed: {
+      classObject: function () {
+        return {
+          'sidebar-hidden': !this.toggleWithoutAnimation && !this.hidden,
+          'sidebar-hidden sidebar-hidden_without-animation': this.toggleWithoutAnimation && !this.hidden
+        }
       }
     }
   }
-
-  function _findMatchingParentMenuItem (itemName) {
-    let parentItem
-    this.menuItems.forEach((item) => {
-      parentItem = (item.children && item.children.find((child) => child.name === itemName)) ? item : parentItem
-    })
-    return parentItem
-  }
 </script>
 
 <style lang="scss">
   @import "../../../sass/_variables.scss";
   @import "~bootstrap/scss/mixins/breakpoints";
   @import "~bootstrap/scss/functions";
   @import "~bootstrap/scss/variables";
-
   .sidebar {
-
     @include media-breakpoint-down(md) {
-
       top: $sidebar-mobile-top;
       left: $sidebar-mobile-left;
       width: $sidebar-mobile-width;
       z-index: $sidebar-mobile-z-index;
     }
-
     height: $sidebar-viewport-height;
     .vuestic-scrollbar {
```

Comment:
Why .sidebar selector prepends .vuestic-sidebar? And what is .sidebar selector? Probably, we don't need it.